src/Resources/config/doctrine.xml

```diff
@@ -16,5 +15,21 @@
             <argument /> <!-- Doctrine's connection name -->
             <argument /> <!-- Doctrine's table name -->
         </service>
+
+        <service id="karo-io.messenger_monitor.storage.stored_message_repository" class="KaroIO\MessengerMonitorBundle\Storage\StoredMessageRepository">
+            <argument type="service" id="karo-io.messenger_monitor.storage.doctrine_connection" />
+            <argument /> <!-- Doctrine's table name -->
+        </service>
```

Comment:
It occurred to me that we may want to organize all the Doctrine classes into their own sub-directory (like in Messenger) instead of organizing them around the bundle. Like, `Storage/Doctrine` with things inside it like `Storage/Doctrine/EventListener`